qtdeploy mkdir panic

This bug appeared in the latest build

```
time="2017-08-23T01:13:49Z" level=panic msg="failed to create dir /go/src/x/ui/lib/deploy/linux/lib" error="mkdir /go/src/x/ui/lib/deploy/linux/lib: not a directory"
panic: (*logrus.Entry) (0x7f9140,0xc42820b590)

goroutine 1 [running]:
github.com/Sirupsen/logrus.Entry.log(0xc42008c0a0, 0xc428809ad0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/user/work/src/github.com/Sirupsen/logrus/entry.go:125 +0x33e
github.com/Sirupsen/logrus.(*Entry).Panic(0xc42820b400, 0xc42da5cf88, 0x1, 0x1)
	/home/user/work/src/github.com/Sirupsen/logrus/entry.go:170 +0x127
github.com/Sirupsen/logrus.(*Entry).Panicf(0xc42820b400, 0x82a8e4, 0x17, 0xc42da5d000, 0x1, 0x1)
	/home/user/work/src/github.com/Sirupsen/logrus/entry.go:218 +0x101
github.com/therecipe/qt/internal/utils.MkdirAll(0xc423386940, 0x35, 0x2, 0xc423386940)
	/home/user/work/src/github.com/therecipe/qt/internal/utils/utils.go:26 +0x131
github.com/therecipe/qt/internal/cmd/deploy.bundle(0x7ffd0a5811c7, 0x5, 0x8036c0, 0x5, 0xc420067890, 0x24, 0xc4200678b1, 0x3, 0xc42006aa40, 0x31)
	/home/user/work/src/github.com/therecipe/qt/internal/cmd/deploy/bundle.go:89 +0x202d
github.com/therecipe/qt/internal/cmd/deploy.Deploy(0x7ffd0a5811c7, 0x5, 0x8036c0, 0x5, 0xc420067890, 0x24, 0x0, 0x7ffd0a581117, 0xaf, 0x0, ...)
	/home/user/work/src/github.com/therecipe/qt/internal/cmd/deploy/deploy.go:60 +0xe4b
main.main()
/home/user/work/src/github.com/therecipe/qt/cmd/qtdeploy/main.go:117 +0x54b
```

--- a/internal/cmd/deploy/bundle.go
+++ b/internal/cmd/deploy/bundle.go
@@ -8,6 +8,7 @@ import (
 	"path/filepath"
 	"runtime"
 	"strings"
+	"time"
 
 	"github.com/therecipe/qt/internal/binding/parser"
 	"github.com/therecipe/qt/internal/binding/templater"
@@ -86,7 +87,8 @@ func bundle(mode, target, path, name, depPath string) {
 				return
 			}
 
-			utils.MkdirAll(filepath.Join(depPath, "lib"))
+			tempLib := "lib" + time.Now().Format("20060102150405")
+			utils.MkdirAll(filepath.Join(depPath, tempLib))
 
 			var (
 				libraryPath   = strings.TrimSpace(utils.RunCmd(exec.Command(utils.ToolPath("qmake", target), "-query", "QT_INSTALL_LIBS"), fmt.Sprintf("query lib path for %v on %v", target, runtime.GOOS)))
@@ -121,7 +123,7 @@ func bundle(mode, target, path, name, depPath string) {
 					}
 
 					if utils.ExistsFile(filepath.Join(libraryPath, libName)) {
-						utils.RunCmd(exec.Command("cp", "-L", filepath.Join(libraryPath, libName), filepath.Join(depPath, "lib", libName)), fmt.Sprintf("copy %v for %v on %v", libName, target, runtime.GOOS))
+						utils.RunCmd(exec.Command("cp", "-L", filepath.Join(libraryPath, libName), filepath.Join(depPath, tempLib, libName)), fmt.Sprintf("copy %v for %v on %v", libName, target, runtime.GOOS))
 					}
 
 					if strings.Contains(dep, "WebEngine") {
@@ -139,11 +141,11 @@ func bundle(mode, target, path, name, depPath string) {
 			}
 			for _, libName := range libs {
 				if utils.ExistsFile(filepath.Join(libraryPath, fmt.Sprintf("libQt5%v.so.5", libName))) {
-					utils.RunCmd(exec.Command("cp", "-L", filepath.Join(libraryPath, fmt.Sprintf("libQt5%v.so.5", libName)), filepath.Join(depPath, "lib", fmt.Sprintf("libQt5%v.so.5", libName))), fmt.Sprintf("copy %v for %v on %v", libName, target, runtime.GOOS))
+					utils.RunCmd(exec.Command("cp", "-L", filepath.Join(libraryPath, fmt.Sprintf("libQt5%v.so.5", libName)), filepath.Join(depPath, tempLib, fmt.Sprintf("libQt5%v.so.5", libName))), fmt.Sprintf("copy %v for %v on %v", libName, target, runtime.GOOS))
 				}
 			}
 			if utils.ExistsFile(filepath.Join(libraryPath, "libqgsttools_p.so.1.0.0")) {
-				utils.RunCmd(exec.Command("cp", "-L", filepath.Join(libraryPath, "libqgsttools_p.so.1.0.0"), filepath.Join(depPath, "lib", "libqgsttools_p.so.1")), fmt.Sprintf("copy libqgsttools_p.so.1 for %v on %v", target, runtime.GOOS))
+				utils.RunCmd(exec.Command("cp", "-L", filepath.Join(libraryPath, "libqgsttools_p.so.1.0.0"), filepath.Join(depPath, tempLib, "libqgsttools_p.so.1")), fmt.Sprintf("copy libqgsttools_p.so.1 for %v on %v", target, runtime.GOOS))
 			}
 
 			libraryPath = filepath.Dir(libraryPath)

